CONTRIBUTING.md

```diff
@@ -215,18 +215,23 @@ Using `docker exec`, this will create a session in the running *awx* container,
 If you want to start and use the development environment, you'll first need to bootstrap it by running the following command:
 
 ```bash
-(container)# /bootstrap_development.sh
+(container)# /usr/bin/bootstrap_development.sh
```

Comment:
Can you open a separate PR for this docs cleanup and minishift removal?

Reply:
done in #6652